Tools/autotest/arduplane.py

```diff
@@ -1124,9 +1124,13 @@ def TestFlaps(self):
 
     def TestRCRelay(self):
         '''Test Relay RC Channel Option'''
-        self.set_parameter("RC12_OPTION", 28) # Relay On/Off
+        self.set_parameters({
+            "RELAY1_FUNCTION": 1, # Enable relay as a standard relay pin
+            "RELAY1_PIN": 4, # Set pin number, looking for 16 in SIM_MASK_PIN
```

Comment:
I don't think this comment is correct... there's a weird one-bit-offset going on?

Reply:
Turns out the test does not actually care what the value is, it just looks for zero and none zero. So we don't need to set the pin at all, just using the default is fine.